project/plugins.sbt

```
@@ -1,10 +1,3 @@
-addSbtPlugin("com.github.sbt" % "sbt-findbugs" % "2.0.0")
-addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.1.0")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.0.0-M2")
 addSbtPlugin("com.typesafe.sbt" % "sbt-osgi" % "0.9.3")
-addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.2")
 
-addSbtPlugin("com.etsy" % "sbt-checkstyle-plugin" % "3.1.1")
-
-addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
-addSbtPlugin("com.eed3si9n" % "sbt-nocomma" % "0.1.0")
+addSbtPlugin("com.github.sbt"    % "sbt-ci-release"     % "1.9.3")
```

```suggestion
addSbtPlugin("com.github.sbt"    % "sbt-ci-release"     % "1.11.1")
```

otherwise you will not be able to publish via the new maven central portal.
and yes you need to update the `SONATYPE_xxx` credentials that you need to generate it https://central.sonatype.com/